### PR TITLE
Add init docs for OpenLab

### PR DESCRIPTION
**Pull request by h00130372**
Add docs for OpenLab, to describe how to join, how to contribute,
and so on.

### CODE REVIEW

File: contributor/Testing_Request.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Testing Request
+The OpenLab provides two kinds of resources: VM(Virtual Machine) and PM(Physical Machine) for testing. Currently we only utilize virtual machines to support automating jobs for SDKs integration test with OpenLab CI. On the other hand you can request PMs for your other testing.
+
+* Submit a test request to OpenLab, fill out some information like what kind of machines (VMs or PMs) and how many do you expect to use, what testing are you planning to implement or need assistance implementing and so on, for more details see [ocean request](https://github.com/theopenlab/ocean/issues/new)
```
Comment (kiwik) [inline contributor/Testing_Request.md]: Please use numbered list to show the step order.
```
1.
2.
3.
```

File: contributor/Connect_to_OpenLab.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Connect to OpenLab
+
+Follow the steps below to Connect the OpenLab:
+
+* OpenLab maintainer will submit an issue to target project that plan to connect OpenLab, let target project maintainers know what we want to do, and discuss some details, looks like: [gophercloud Issue 592](https://github.com/gophercloud/gophercloud/issues/592)
```
Comment (kiwik) [inline contributor/Connect_to_OpenLab.md]: Please use numbered list to show the step order.
```
1.
2.
3.
```

File: Member_Application.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Member Application
+To be a member of OpenLab follow the simple steps below:
+
+* Submit an issue to [apply to join](https://github.com/theopenlab/joinin/issues/new) first, fill your basic information, the way you will support to OpenLab and so on
```
Comment (kiwik) [inline Member_Application.md]: Please use numbered list to show the step order.
```
1.
2.
3.
```

File: README.md [hunk 4]
```diff
@@ -5,6 +5,25 @@ description: >-
...
+Join us to be a member of OpenLab firstly, then you will find out what's happening in OpenLab, what fellow open source community members are doing using OpenLab resources, also as a member you can access OpenLab resources for your testing, please follow the simple [member application](Member_Application.md) guide.
+
+### Test Request
+If you wish to use OpenLab resources, start your test request [here](./contributor/Testing_Request.md).
+
+### Development
+The main work of OpenLab CI is to provide an auto integration testing for different upstream tool chains of OpenStack, e.g. Gophercloud, Terraform. Always we have to add CI jobs for the test requests submitted to OpenLab, if you are interesting in this job, please follow [Development Guide](contributor/Development_Workflow.md).
+
+### Resource sponsoring
+The OpenLab has multiple resource pools provided by multiple cloud providers to run testing jobs. Thanks to our sponsors we also can offer the dedicated resources to users for their own tests. We'd very appreciate it if you would like to [donate](./contributor/Resource_Sponsor.md) resources to us.
```
Comment (kiwik) [inline README.md]: 1. multiple resource pools(virtual machine, physical machine and device)
2. run testing jobs, like: VEXXHOST, OpenTelekomCloud, Citynetwork and so on.
3. resources to OpenLab.

File: README.md [hunk 5]
```diff
@@ -5,6 +5,25 @@ description: >-
...
 
 ## Submitting Your Publication
```
Comment (kiwik) [inline README.md]: I prefer to remove this whole section `Submitting Your Publication`, this part content have been included in `Test Request` section

File: README.md [hunk 6]
```diff
@@ -15,5 +34,11 @@ In order to get started you will want to do a couple of things.
 * clone this document repository
 * create a branch with your request ID \(issue number provided when you requested resources\)
 
+## References
+* OpenLab Web Site is hosted on [OpenLab](https://openlabtesting.org/)
```
Comment (kiwik) [inline README.md]: s/OpenLab Web Site/OpenLab web site/

File: README.md [hunk 7]
```diff
@@ -15,5 +34,11 @@ In order to get started you will want to do a couple of things.
...
+* Join our IRC Channel `#askopenlab` on freenode.net
```
Comment (kiwik) [inline README.md]: s/Channel/channel/

Comment (kiwik) [reply README.md]: Add a link at the end, not everyone is familiar how to connect to freenode [See](https://freenode.net/kb/answer/chat)

File: README.md [hunk 8]
```diff
@@ -15,5 +34,11 @@ In order to get started you will want to do a couple of things.
...
+* The OpenLab CI github APP is hosted on [github APP](https://github.com/apps/theopenlab-ci)
+* Source code of OpenLab CI jobs is hosted on [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs)
+* CI Jobs' status: [Status](http://status.openlabtesting.org/t/openlab/status.html)
```
Comment (kiwik) [inline README.md]:
```
OpenLab CI Jobs' [status](http://status.openlabtesting.org/t/openlab/status.html)
```

File: README.md [hunk 9]
```diff
@@ -15,5 +34,11 @@ In order to get started you will want to do a couple of things.
 * clone this document repository
 * create a branch with your request ID \(issue number provided when you requested resources\)
 
+## References
+* OpenLab Web Site is hosted on [OpenLab](https://openlabtesting.org/)
+* Join our IRC Channel `#askopenlab` on freenode.net
+* The OpenLab CI github APP is hosted on [github APP](https://github.com/apps/theopenlab-ci)
```
Comment (kiwik) [inline README.md]: I think we should switch the order of `github APP` and `CI jobs' status` in the list, let newbie to get interesting content at first

File: README.md [hunk 10]
```diff
@@ -5,6 +5,25 @@ description: >-
...
+There is plenty of room for helping the shape of future of OpenLab and ensuring things run smoothly. Welcome to join OpenLab if you're interested in our system, please follow the guide below:
+
+### To be an OpenLab member
+Join us to be a member of OpenLab firstly, then you will find out what's happening in OpenLab, what fellow open source community members are doing using OpenLab resources, also as a member you can access OpenLab resources for your testing, please follow the simple [member application](Member_Application.md) guide.
+
+### Test Request
+If you wish to use OpenLab resources, start your test request [here](./contributor/Testing_Request.md).
+
+### Development
+The main work of OpenLab CI is to provide an auto integration testing for different upstream tool chains of OpenStack, e.g. Gophercloud, Terraform. Always we have to add CI jobs for the test requests submitted to OpenLab, if you are interesting in this job, please follow [Development Guide](contributor/Development_Workflow.md).
```
Comment (kiwik) [inline README.md]: s/[Development Guide]/[development guide]/

File: contributor/Testing_Request.md [hunk 11]
```diff
@@ -0,0 +1,7 @@
+# Testing Request
+The OpenLab provides two kinds of resources: VM(Virtual Machine) and PM(Physical Machine) for testing. Currently we only utilize virtual machines to support automating jobs for SDKs integration test with OpenLab CI. On the other hand you can request PMs for your other testing.
+
+* Submit a test request to OpenLab, fill out some information like what kind of machines (VMs or PMs) and how many do you expect to use, what testing are you planning to implement or need assistance implementing and so on, for more details see [ocean request](https://github.com/theopenlab/ocean/issues/new) 
+* The governance team owner will confirm the request to check whether it is acceptable, if yes then will add it to [TODO List](https://github.com/orgs/theopenlab/projects/1#column-1860008)
+* The test request in TODO list can be assigned, if you wish to work on it please pick it up, move the item to [IN PROGRESS List](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit the item to add the development plan
+* Begin your development and test if you picked up the ocean request, and follow [Development Guide](./Development_Workflow.md)
```
Comment (kiwik) [inline contributor/Testing_Request.md]: Add an item about "Share your testing result", include:
1. If you use OpenLab CI, the test result will be shown in http://status.openlabtesting.org/builds automatically
2. If you use physical machine to test, please commit the test result into comments of testing request related issue. A blog also is a good way to show more details of testing.

File: README.md [hunk 12]
```diff
@@ -5,6 +5,25 @@ description: >-
...
+
+## How to Join OpenLab
+
+There is plenty of room for helping the shape of future of OpenLab and ensuring things run smoothly. Welcome to join OpenLab if you're interested in our system, please follow the guide below:
+
+### To be an OpenLab member
+Join us to be a member of OpenLab firstly, then you will find out what's happening in OpenLab, what fellow open source community members are doing using OpenLab resources, also as a member you can access OpenLab resources for your testing, please follow the simple [member application](Member_Application.md) guide.
+
+### Test Request
+If you wish to use OpenLab resources, start your test request [here](./contributor/Testing_Request.md).
```
Comment (kiwik) [inline README.md]: If you have an idea or real use case about some integration scenarios, like: integrating Kubernetes and OpenStack, plan to do proof of concept(POC) against some technologies, all you have to do is just sharing your idea and case in OpenLab, if others are interested on your case, they maybe pick up your task and complete the subsequent testing works, pay attention on updating of OpenLab [web site](https://openlabtesting.org/explore/), we will share the test result on it. Start your test request [here](./contributor/Testing_Request.md) now.

File: Member_Application.md (hunk 3 above)
Comment (mrhillsman) [inline Member_Application.md]:
```suggestion
* Open an issue to [apply to join](https://github.com/theopenlab/ocean/issues/new/choose) first, fill out the form completely with as much detail, for example the way you will support to OpenLab
```

File: Member_Application.md [hunk 13]
```diff
@@ -0,0 +1,7 @@
+# Member Application
```
Comment (mrhillsman) [inline Member_Application.md]: What is this related to? Member application for using resources?

File: Member_Application.md [hunk 14]
```diff
@@ -0,0 +1,7 @@
...
+To be a member of OpenLab follow the simple steps below:
+
+* Submit an issue to [apply to join](https://github.com/theopenlab/joinin/issues/new) first, fill your basic information, the way you will support to OpenLab and so on
+* The governance team owner will assign a role to you based on your information, then will contact you and send you an invitation email, please confirm it and reply to us
+* Then you are the member of OpenLab, and the application issue will be closed by governance team owner
+* Welcome to the playground! As a member of OpenLab, do what you want using the OpenLab resources!
```
Comment (mrhillsman) [inline Member_Application.md]:
```suggestion
* Welcome to the playground! As a member of OpenLab, ~~do what you want using the OpenLab resources~~!
```

File: README.md [hunk 15]
```diff
@@ -5,6 +5,25 @@ description: >-
 ---
 
 # Introduction
+## OpenLab
+
+OpenLab is a community-led program to test and improve support for the most popular Software Development Kits(SDKs) as well as platforms like Kubernetes, Terraform, CloudFoundry and more. The goal is to improve the usability, reliability and resiliency of tools and applications for hybrid and multi-cloud environments.
+
+## How to Join OpenLab
+
+There is plenty of room for helping the shape of future of OpenLab and ensuring things run smoothly. Welcome to join OpenLab if you're interested in our system, please follow the guide below:
```
Comment (mrhillsman) [inline README.md]:
```suggestion
There is plenty of room for helping to shape of future of OpenLab and ensuring things run smoothly. If you are interested in our system, please follow the guidelines below:
```

File: Member_Application.md (hunk 3 above)
Comment (h00130372) [reply Member_Application.md]: we still have to discuss the repo where to open the issue.

File: contributor/Member_Application.md [hunk 16]
```diff
@@ -0,0 +1,8 @@
+# Member Application
+To be a member of OpenLab follow the simple steps below:
+
+1. Open an issue to [apply to join](https://github.com/theopenlab/openlab/issues/new?template=joinin.md) first, fill out the form completely with as much detail, for example your contact information and the way you will support to OpenLab
+2. The governance team owner will assign a role to you based on your information, then will contact you and send you an invitation email, please confirm it and reply to us
```
Comment (kiwik) [inline contributor/Member_Application.md]: The OpenLab governance team owner will join you into [team](https://github.com/orgs/theopenlab/teams) based on your information, then will contact you and send you an invitation in github.com, please pay attention on Github [notifications](https://github.com/notifications) and confirm it.

File: contributor/Member_Application.md [hunk 17]
```diff
@@ -0,0 +1,8 @@
...
+3. Then you are the member of OpenLab, please visit the OpenLab project [status board](https://github.com/orgs/theopenlab/projects/1) to get the overview of what's going on
+4. The application issue will be closed by governance team owner
+5. Welcome to the playground!
```
Comment (kiwik) [inline contributor/Member_Application.md]: 4. Welcome to the playground! Say "Hi" in OpenLab IRC channel `#askopenlab` on freenode.net :-)
5. The application issue will be closed by governance team owner

File: contributor/Testing_Request.md [hunk 18]
```diff
@@ -0,0 +1,11 @@
+# Testing Request
+The OpenLab provides two kinds of resources: VM(Virtual Machine) and PM(Physical Machine) for testing. Currently we only utilize virtual machines to support automating jobs for SDKs integration test with OpenLab CI. On the other hand you can request PMs for your other testing.
```
Comment (kiwik) [inline contributor/Testing_Request.md]: SDKs and tools integration

File: contributor/Development_Workflow.md [hunk 19]
```diff
@@ -0,0 +1,26 @@
+# Development Workflow
+
+## Prerequisites
+The OpenLab CI is built based on `NodePool` and `Zuul` tools of the OpenStack infrastructure. The CI Jobs definition uses Ansible playbooks. Learn more about the tools first:
+* Zuul documentation is hosted on [Zuul](https://zuul-ci.org)
+* NodePool documentation is hosted on [NodePool](https://zuul-ci.org/docs/nodepool/)
+* Ansible documentation is hosted on [Ansible Docs](https://docs.ansible.com/)
+
+## Working on Ocean request
+The accepted test request items can be found on [TODO List](https://github.com/orgs/theopenlab/projects/1#column-1860008).
```
Comment (liu-sheng) [inline contributor/Development_Workflow.md]: Developers can get start with following steps:
1...
2..
3...

File: contributor/Development_Workflow.md [hunk 20]
```diff
@@ -0,0 +1,26 @@
...
+1. Pick one item you wish to work on, move it to [IN PROGRESS List](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit the item to add the development plan
```
Comment (liu-sheng) [inline contributor/Development_Workflow.md]: Pike an item you want to work on, and move it to the **[IN PROGRESS](https://github.com/orgs/theopenlab/projects/1#column-1860011)** list, you can also feel free to edit the item according to your development plan.

**Comment by kiwik** [comment]
TODO list:
1. Update deprecated README https://github.com/theopenlab/openlab-zuul-jobs#more-information
2. Update deprecated describe https://github.com/apps/theopenlab-ci
3. Please check OpenLab documents whether link to deprecate pages.

File: contributor/Testing_Request.md [hunk 21]
```diff
@@ -0,0 +1,11 @@
+# Testing Request
+The OpenLab provides two kinds of resources: VM(Virtual Machine) and PM(Physical Machine) for testing. Currently we only utilize virtual machines to support automating jobs for SDKs and tools integration test with OpenLab CI. On the other hand you can request PMs for your other testing.
+
+1. Submit a test request to OpenLab, fill out some information like what kind of machines (VMs or PMs) and how many do you expect to use, what testing are you planning to implement or need assistance implementing and so on, for more details see [test request](https://github.com/theopenlab/openlab/issues/new?template=integration_test_request.md) 
+2. The governance team will confirm the request to check whether it is acceptable, if yes then will add it to [TODO list](https://github.com/orgs/theopenlab/projects/1#column-1860008)
+3. The test request in TODO list can be assigned, if you wish to work on it please pick it up, move the item to [IN PROGRESS list](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit the item to add the development plan
```
Comment (kiwik) [inline contributor/Testing_Request.md]: 1. s/if you wish/if you or another OpenLab members wish/
2. s/it/the item/
3. s/edit the item to add the development plan/edit it to add the development plan into comments/

File: contributor/Testing_Request.md [hunk 22]
```diff
@@ -0,0 +1,11 @@
...
+4. Begin your development and test if you picked up the ocean request, and follow [development guide](./Development_Workflow.md)
```
Comment (kiwik) [inline contributor/Testing_Request.md]: s/ocean request/test request/

File: contributor/Testing_Request.md [hunk 23]
```diff
@@ -0,0 +1,11 @@
...
+
+# Share your testing result
+* If you use OpenLab CI, the test result will be shown on http://status.openlabtesting.org/builds automatically
+* If you use physical machine to test, please commit the test result into comments of testing request related issue. A blog also is a good choice to show more details of testing.
```
Comment (kiwik) [inline contributor/Testing_Request.md]: s/testing request/test request/

File: contributor/Development_Workflow.md [hunk 24]
```diff
@@ -0,0 +1,26 @@
+# Development Workflow
```
Comment (kiwik) [inline contributor/Development_Workflow.md]: 1. s/Development Workflow/Development guide/
2. file name should be `Development_guide.md`

File: contributor/Development_Workflow.md [hunk 25]
```diff
@@ -0,0 +1,26 @@
...
+
+## Prerequisites
+The OpenLab CI is built based on `NodePool` and `Zuul` tools of the OpenStack infrastructure. The CI Jobs definition uses Ansible playbooks. Learn more about the tools first:
+* Zuul documentation is hosted on [Zuul](https://zuul-ci.org)
+* NodePool documentation is hosted on [NodePool](https://zuul-ci.org/docs/nodepool/)
+* Ansible documentation is hosted on [Ansible Docs](https://docs.ansible.com/)
+
+## Working on Ocean request
```
Comment (kiwik) [inline contributor/Development_Workflow.md]: s/Ocean request/Test request/

File: contributor/Development_Workflow.md [hunk 26]
```diff
@@ -0,0 +1,26 @@
+# Development Workflow
+
+## Prerequisites
+The OpenLab CI is built based on `NodePool` and `Zuul` tools of the OpenStack infrastructure. The CI Jobs definition uses Ansible playbooks. Learn more about the tools first:
+* Zuul documentation is hosted on [Zuul](https://zuul-ci.org)
+* NodePool documentation is hosted on [NodePool](https://zuul-ci.org/docs/nodepool/)
+* Ansible documentation is hosted on [Ansible Docs](https://docs.ansible.com/)
+
```
Comment (kiwik) [inline contributor/Development_Workflow.md]: Please add pictures of OpenLab CI arch and communication with Github at here.

File: contributor/Development_Workflow.md (hunk 20 above)
Comment (kiwik) [reply contributor/Development_Workflow.md]: s/one item/one test request/

File: contributor/Development_Workflow.md [hunk 27]
```diff
@@ -0,0 +1,26 @@
...
+## Working on Ocean request
+The accepted test request items can be found on [TODO List](https://github.com/orgs/theopenlab/projects/1#column-1860008).
+1. Pick one item you wish to work on, move it to [IN PROGRESS List](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit the item to add the development plan
+2. The target project of the request item should install the **OpenLab-CI** [github APP](https://github.com/apps/theopenlab-ci), so that OpenLab CI system and target project can communicate with each other. See the details on [connecting guideline](./Connect_to_OpenLab.md)
```
Comment (kiwik) [inline contributor/Development_Workflow.md]: s/request item/test request/

File: contributor/Development_Workflow.md [hunk 28]
```diff
@@ -0,0 +1,26 @@
...
+3. Fork your own copy of the CI Job repository [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs) to your account, develop the test job and submit a PR(Pull Request). In order to manage the test request well, please relate the request number in PR's commit message, the format like:
```
Comment (kiwik) [inline contributor/Development_Workflow.md]: 1. Fork OpenLab CI Jobs repository [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs) to your account
2. submit a PR(Pull Request) to [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs)
3. please add change details and link the test request number in PR's commit message

File: contributor/Development_Workflow.md [hunk 29]
```diff
@@ -0,0 +1,26 @@
...
+6. Close the `In Progress` item yourself or contact our development core to close it when the request test complete 
+
+## Working on Bug
+1. Bug reports for OpenLab are tracked on [bug list](https://github.com/theopenlab/openlab/issues). 
+2. The merged CI job will fail due to some reasons after it has been running for some time, or if you find any other problems of OpenLab welcome to [new](https://github.com/theopenlab/openlab/issues/new?template=bug_report.md) an issue
+3. The governance team will confirm the bugs, add them to [TODO list](https://github.com/orgs/theopenlab/projects/1#column-1860008)
+4. You can assign one bug from TODO list, move the item to [IN PROGRESS list](https://github.com/orgs/theopenlab/projects/1#column-1860011)
```
Comment (kiwik) [inline contributor/Development_Workflow.md]:
```suggestion
4. You can assign one bug from TODO list, move the bug to [IN PROGRESS list](https://github.com/orgs/theopenlab/projects/1#column-1860011) . Feel free to pick up the issues with label [need volunteer](https://github.com/theopenlab/openlab/issues?q=is%3Aissue+is%3Aopen+label%3A%22need+volunteer%22), these issues are not claimed by anyone yet.
```

File: contributor/Development_Workflow.md [hunk 30]
```diff
@@ -0,0 +1,26 @@
...
+6. Close the `In Progress` item yourself or contact our development core to close it when the request test complete 
+
+## Working on Bug
+1. Bug reports for OpenLab are tracked on [bug list](https://github.com/theopenlab/openlab/issues). 
+2. The merged CI job will fail due to some reasons after it has been running for some time, or if you find any other problems of OpenLab welcome to [new](https://github.com/theopenlab/openlab/issues/new?template=bug_report.md) an issue
+3. The governance team will confirm the bugs, add them to [TODO list](https://github.com/orgs/theopenlab/projects/1#column-1860008)
+4. You can assign one bug from TODO list, move the item to [IN PROGRESS list](https://github.com/orgs/theopenlab/projects/1#column-1860011) 
+5. Submit a PR(Pull Request), no matter where the PR submitted to, the issue number should be related in commit message. And then do the test to fix the bug
```
Comment (kiwik) [inline contributor/Development_Workflow.md]:
```suggestion
5. Submit a PR(Pull Request) to [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs) to fix bug,  please add change details and link the bug number in PR's commit message
`Related-Bug: theopenlab/openlab#number`
6. OpenLab development team @theopenlab/dev will review the PR and give some suggestion, development core team @theopenlab/dev-core have permission to merge it if there is no issue. When the PR is merged into openlab-zuul-jobs, you can test jobs by [trigger comments](./Supported_trigger_comments.md).
```

File: contributor/Development_Workflow.md [hunk 31]
```diff
@@ -0,0 +1,26 @@
...
+6. Close the `In Progress` item yourself or contact our development core to close it when the bug is fixed
```
Comment (kiwik) [inline contributor/Development_Workflow.md]:
```suggestion
7. Close the `In Progress` bug by yourself when the bug is fixed.
```

File: contributor/Connect_to_OpenLab.md [hunk 32]
```diff
@@ -0,0 +1,7 @@
+# Connect to OpenLab
+
+Follow the steps below to Connect the OpenLab:
+
+1. OpenLab maintainer will submit an issue to target project that plan to connect OpenLab, let target project maintainers know what we want to do, and discuss some details, looks like: [gophercloud Issue 592](https://github.com/gophercloud/gophercloud/issues/592)
+2. If the above issue is approved, OpenLab maintainer will commit a pull request to target project, **.zuul.yaml** will be included in PR, that define which CI jobs will run, looks like: [gophercloud PR 593](https://github.com/gophercloud/gophercloud/pull/593)
+3. Target project maintainer should install **TheOpenLab-CI** [github APP](https://github.com/apps/theopenlab-ci), so that OpenLab CI system and target project can communicate with each other, **TheOpenLab-CI** github APP only require basic permissions to update check status and commit test result log URL in PR comments, you can see permission details in APP installation web page.
```
[the remark this answers is not in the document]
Comment (h00130372) [reply contributor/Connect_to_OpenLab.md]: no need to modify this after discuss:)

File: contributor/Development_Workflow.md (hunk 28 above)
Comment (h00130372) [reply contributor/Development_Workflow.md]: addressed 2 & 3.

File: contributor/Resource_Sponsor.md [hunk 33]
```diff
@@ -0,0 +1,6 @@
+# How to sponsor resources
+All of resources that are donated to OpenLab list in [here](https://github.com/theopenlab/resources). Now we have many cloud providers to provide VMs for CI testing, and also we have some dedicated resources donated by our sponsors. If you would like to provide your cloud or other kind of resources, read the guide bellow:
+1. New an [issue](https://github.com/theopenlab/openlab/issues/new?template=resource_donating.md) to describe what kind and how many resources you would like to provide, fill out the form to tell the details including your contact information and so on
+2. After received your donation request, our marketing member will contact you as soon as possible
```
Comment (kiwik) [inline contributor/Resource_Sponsor.md]:
```suggestion
2. After received your donation information, OpenLab governance member @theopenlab/governance will contact you as soon as possible   
```

File: contributor/Resource_Sponsor.md [hunk 34]
```diff
@@ -0,0 +1,6 @@
+# How to sponsor resources
+All of resources that are donated to OpenLab list in [here](https://github.com/theopenlab/resources). Now we have many cloud providers to provide VMs for CI testing, and also we have some dedicated resources donated by our sponsors. If you would like to provide your cloud or other kind of resources, read the guide bellow:
+1. New an [issue](https://github.com/theopenlab/openlab/issues/new?template=resource_donating.md) to describe what kind and how many resources you would like to provide, fill out the form to tell the details including your contact information and so on
+2. After received your donation request, our marketing member will contact you as soon as possible   
+3. The marketing member will ask operator to add the cloud provider or other kind of resources to OpenLab CI system
```
Comment (kiwik) [inline contributor/Resource_Sponsor.md]:
```suggestion
3. The OpenLab governance member will ask operator team @theopenlab/ops to add the cloud provider or other kind of resources to OpenLab CI or dedicated resource pool when donation resources are availability.
```

File: contributor/Resource_Sponsor.md [hunk 35]
```diff
@@ -0,0 +1,6 @@
...
+4. Last the operator will close the donation request. Thank you very much for your donation!
```
Comment (kiwik) [inline contributor/Resource_Sponsor.md]:
```suggestion
4. At last @theopenlab/ops will close the donation when everything is OK. Thank you very much for your donation! 
```

File: README.md [hunk 36]
```diff
@@ -5,15 +5,29 @@ description: >-
 ---
 
 # Introduction
+## OpenLab
 
-## Submitting Your Publication
+OpenLab is a community-led program to test and improve support for the cloud-related SDKs/Tools as well as platforms like Kubernetes, Terraform, CloudFoundry and more. The goal is to improve the usability, reliability and resiliency of tools and applications for hybrid and multi-cloud environments.
```
Comment (kiwik) [inline README.md]: I think if we can add a step drawing at here to describe the order of `To be an OpenLab member`, `Test request`, `Development` and `Resource sponsoring`, it is better.

File: contributor/Development_guide.md [hunk 37]
```diff
@@ -0,0 +1,41 @@
...
+   
+   `Related-Bug: theopenlab/openlab#number` 
+4. OpenLab development team @theopenlab/dev will review the PR and give some suggestion, development core team @theopenlab/dev-core have permission to merge it if there is no issue, at the meanwhile, you should contact @theopenlab/dev-core to add the target project into **main.yaml** in **zuul** node if it is not in project list of zuul
+5. When your PR is merged into master of [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs), trigger to test the integration job by [trigger comments](./Supported_trigger_comments.md), see the [build status](http://status.openlabtesting.org/t/openlab/builds.html), click `logs` to get the log details if the job fail. Maybe you can find some problems of job, feel free to submit new pull request to [openlab-zuul-jobs](https://github.com/theopenlab/openlab-zuul-jobs/pulls) to fix job issue, link the issue number of related test request in PR's commit message as we do on step3
```
Comment (kiwik) [inline contributor/Development_guide.md]: We apply new release portal of OpenLab CI after latest monthly upgrading, so please update this describe to match new links, and `logs` have been move into STATUS link.

File: README.md [hunk 38]
```diff
@@ -5,15 +5,29 @@ description: >-
...
 
+### Development
+The main work of OpenLab is to provide an auto integration testing for different upstream tool chains of OpenStack, e.g. Gophercloud, Terraform. Always we have to add CI jobs for the test requests submitted to OpenLab, if you are interesting in this job, please follow [development guide](contributor/Development_guide.md).
 
+### Resource sponsoring
+The OpenLab has multiple resource pools(virtual machine, physical machine and device) provided by multiple cloud providers to run testing jobs, like VEXXHOST, OpenTelekomCloud, Citynetwork and so on. Thanks to OpenLab sponsors we also can offer the dedicated resources to users for their own tests. We'd very appreciate it if you would like to [donate](./contributor/Resource_Sponsor.md) resources to OpenLab.
```
Comment (kiwik) [inline README.md]:
```suggestion
The OpenLab has multiple resource pools(virtual machine, physical machine and device) provided by multiple cloud providers to run testing jobs, like VEXXHOST, OpenTelekomCloud, Citynetwork and so on. Thanks to OpenLab sponsors we can offer both virtual machine and physical machine to users for their own tests. We'd very appreciate it if you would like to [donate](./contributor/Resource_Sponsor.md) resources to OpenLab.
```

File: contributor/Resource_Sponsor.md [hunk 39]
```diff
@@ -0,0 +1,7 @@
+# How to sponsor resources
+We list all of resources that are donated to OpenLab in [website](https://openlabtesting.org/explore/). Now we have many cloud providers to provide VMs for CI testing, and also we have some dedicated resources donated by our sponsors. If you would like to provide your cloud or other kind of resources, read the guide bellow:
+
+1. Open an [issue](https://github.com/theopenlab/openlab/issues/new?template=resource_donating.md&labels=donating) to describe what kind and how many resources you would like to provide, fill out the form to tell the details
+2. After received your donation information, OpenLab governance member @theopenlab/governance will contact you as soon as possible   
+3. The OpenLab governance member will ask operator team @theopenlab/ops to add the cloud provider or other kind of resources to OpenLab CI or dedicated resource pool when donation resources are available
```
Comment (kiwik) [inline contributor/Resource_Sponsor.md]: s/add the cloud provider or other kind of resources/add virtual machine or other kind of resources/

File: contributor/Testing_Request.md [hunk 40]
```diff
@@ -0,0 +1,11 @@
+# Test request
+The OpenLab provides two kinds of resources: VM(Virtual Machine) and PM(Physical Machine) for testing. Currently we only utilize virtual machines to support automating jobs for SDKs and tools integration test with OpenLab CI. On the other hand you can request PMs for your other testing.
+
+1. Submit a [test request](https://github.com/theopenlab/openlab/issues/new?template=integration_test_request.md) to OpenLab, fill out some information to show your test cases like what testing scenario are you planning to implement, what kind of machines (VMs or PMs) and how many do you expect to use and so on 
+2. The governance team will confirm the request to check whether it is acceptable, if yes then will add it to [TODO list](https://github.com/orgs/theopenlab/projects/1#column-1860008)
+3. The test request in TODO list can be assigned, if you wish to work on it please pick it up, move it to [IN PROGRESS list](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit it to add the development plan into comments
```
[the remark this answers is not in the document]
Comment (h00130372) [reply contributor/Testing_Request.md]: Here I think 'you' is enough, it's general reference here.

**Pull request merged**